### ISSUE TITLE
Package.json Homepage refers to incorrect Github page

### ISSUE THREAD

**Issue by raybooysen** [react-component/collapse#24]
Package.json homepage refers to incorrect URL. 

```
  "homepage": "http://github.com/react-component/rc-collapse",
```

**Comment by benjycui** [comment]
THX! And you had better pull request next time :-)

**Comment by raybooysen** [comment]
Haha sure thing. :)

Closed #24 <https://github.com/react-component/collapse/issues/24>.

—
Reply to this email directly or view it on GitHub
<https://github.com/react-component/collapse/issues/24#event-550127820>.
